src/common/MessageDispatcher.sol

```diff
@@ -641,7 +641,6 @@ contract MessageDispatcher is Auth, IMessageDispatcher {
         PoolId poolId,
         ShareClassId scId,
         AssetId assetId,
-        address provider,
```

Comment:
This param seems unused entirely. @mustermeiszer do you agree it can be removed?

Reply:
Yes